src/tooltip.js

```diff
@@ -1,440 +1,327 @@
-/* Tooltip rendering model for nvd3 charts.
- window.nv.models.tooltip is the updated,new way to render tooltips.
 
- window.nv.tooltip.show is the old tooltip code.
- window.nv.tooltip.* also has various helper methods.
+/* Model which can be instantiated to handle tooltip rendering.
+ Example usage:
+ var tip = nv.models.tooltip().gravity('w').distance(23)
+ .data(myDataObject);
+
+ tip();    //just invoke the returned function to render tooltip.
  */
-(function() {
+nv.models.tooltip = function() {
```

Comment:
Well, of course. Every time you call nv.models.tooltip() you're asking for a new tooltip. If you want to use the same tooltip you just save the tooltip you created and use it. This is the same behavior of all the other models.